fs-person/fs-person.html

```diff
@@ -27,6 +28,11 @@
       :host([inline]) .fs-person-vitals__name {
         margin-right: 15px;
       }
+      .fs-person-vitals,
+      .fs-person-vitals__name-block {
+        width: 98%;
+        width: calc(100% - 1.143rem);
```

Comment:
Is there a different way of solving this issue? This approach seems fragile to me.